pkg/sqlite/sql.go

```diff
@@ -218,6 +223,11 @@ func getMultiCriterionClause(primaryTable, foreignTable, joinTable, primaryFK, f
 	return whereClause, havingClause
 }
 
+func getCountCriterionClause(primaryTable, joinTable, primaryFK string, criterion models.IntCriterionInput) (string, int) {
+	lhs := fmt.Sprintf("(SELECT COUNT(*) FROM %s s WHERE s.%s = %s.id)", joinTable, primaryFK, primaryTable)
```

Comment:
I believe you are missing the `IFNULL(..., 0)` around the query, as I had originally. Without that I believe it won't work properly for count = 0 condition.